packages/ui/src/coreModules/resourceManager/components/ResourceManager/MainColumn/item/ActionBars/higherOrderComponents/createHandleCreateSubmit/index.js

```diff
@@ -73,12 +79,18 @@ const createHandleCreateSubmit = () => ComposedComponent => {
         create({
           item: transformOutput ? transformOutput(values) : values,
           nested: true,
+          throwOnError: true,
```

Comment:
it looks like true is the default? I think either omit it here or also set it explicitly in createHandleEditSubmit would be more consistent. now I had to have a look to see if they were different or if it was a bug that true was not set in edit

Reply:
sure